packages/shared/src/components/LoginButton.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { ReactElement, useContext, useEffect } from 'react';
+import { Button } from './buttons/Button';
+import AuthContext from '../contexts/AuthContext';
+import FeaturesContext from '../contexts/FeaturesContext';
+import { getFeatureValue } from '../lib/featureManagement';
+import AnalyticsContext from '../contexts/AnalyticsContext';
+import { AnalyticsEvent } from '../hooks/analytics/useAnalyticsQueue';
+
+const getAnalyticsEvent = (
+  eventName: string,
+  copy: string,
+): Partial<AnalyticsEvent> => ({
+  event_name: eventName,
+  target_type: 'signup button',
+  target_id: 'header',
+  feed_item_title: copy,
+});
+
+export default function LoginButton(): ReactElement {
+  const { showLogin } = useContext(AuthContext);
+  const { flags } = useContext(FeaturesContext);
+  const { trackEvent } = useContext(AnalyticsContext);
+  const buttonCopy = getFeatureValue('signup_button_copy', flags) || 'Login';
```

Comment:
Wow, didn't we know we have this function. I will for sure use it next time.

I was thinking now to create an `enum` (or `type` which we can use as a parameter type for the function itself) or an `object` that will have these strings so it can be much easier to use on some other parts and ensure to have the same string that is being called. What do you think?

Reply:
This is a great suggestion. I had auto merge turned on so I'll push another branch